frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -638,17 +638,17 @@ Except when in two columns mode, where this is limited to showing only the previ
         table.insert(menu_items.long_press.sub_item_table, {
             text_func = function()
                 if G_reader_settings:readSetting("highlight_non_touch_interval") == 1 then
-                    return T(_("Interval to speed-up rate: %1 second"), G_reader_settings:readSetting("highlight_non_touch_interval", 1))
+                    return T(_("Interval to speed-up rate: %1 second"), G_reader_settings:readSetting("highlight_non_touch_interval") or 1)
                 else
-                    return T(_("Interval to speed-up rate: %1 seconds"), G_reader_settings:readSetting("highlight_non_touch_interval", 1))
+                    return T(_("Interval to speed-up rate: %1 seconds"), G_reader_settings:readSetting("highlight_non_touch_interval"))
```

Comment:
This may display `rate: nil seconds` or `rate: seconds` when the setting is not set ? so needs a `or defaultvalue`.

May be @Frenzie will want to teach you the pleasure of the `T()` equivalent that deal with international plurals :)